specs.tex

```diff
@@ -66,6 +66,11 @@ \subsection{Arena}
         that of other zones.
   \item Tokens will be placed in the scoring zone on the opposite side to their
         matching coloured scoring zone.
+  \item \label{spec:flags} Flags will be made from wire folded in half
```

Comment:
why is the folding in half relevant? I'm not sure I understand what it is that we're trying to describe with this.